lib/ansible/modules/packaging/language/pip.py

```diff
@@ -519,13 +652,12 @@ def main():
                 extra_args = ' '.join(args_list)
 
         if extra_args:
-            cmd += ' %s' % extra_args
+            cmd.append(extra_args)
 
         if name:
-            for pkg in name:
-                cmd += ' %s' % _get_full_name(pkg, version)
+            cmd.extend(str(p) for p in packages)
```

Comment:
I believe @abadger meant this place. So you basically want to use `to_native()` instead.